pages/docs/advanced/meta.en-US.json

```diff
@@ -1,4 +1,5 @@
 {
+  "return-values": "Return Values",
```

Comment:
Shall we rename this to `Understanding SWR` or `State Machine`?

And add a link from api page to this?

Reply:
got inspired by redux docs
<img width="294" alt="image" src="https://user-images.githubusercontent.com/4800338/188272071-94cb470c-3373-4356-b91e-62cdec5bb887.png">

Reply:
That's great! I'll rename this to `Understanding SWR`, and the section to `State Machine`.